### PR TITLE
[DDW-680] Small UI/UX fixes

### PR DESCRIPTION
**Pull request by mchappell**
This PR makes a number of small UI/UX across the Daedalus application

## Todos

- [x] Stake pool tooltip should say 'Copied' while checkmark confirmation is visible
- [x] Padding for share wallet adjusted
- [x] 'Back to top' button in stake pool list should only appear after first row of pools is over scrolled
- [x] Countdown elements should always fit on one line
- [x] Staking pools copy adjusted
- [x] Blocks synced tooltip matches design of other tooltips
- [x] Wallet name should always fit on a single line, trimmed with '...'
- [x] Disabled state for red checkbox in send dialog has 0.3 opacity
- [x] 'Copy address' should have hover effect for text and icon
- [x] Confirm transaction address uses 100% opacity
- [x] Hover effect for wallet public key triggers hover state for copy button
- [x] Make progress bars animation consistent
- [x] Align delegation center loading spinners to right of container
- [x] Align wallet recovery step indicators
- [x] Adjust font for off-chain pool information
- [x] Increase drop shadow in staking pools cards
- [x] Change wallet restoration text
- [x] Show newly restored wallets immediately in wallet list
- [x] Adjust distance between name and balance in delegation center
- [x] Adjust font color for recovery phrase indicator of redemption rewards
- [x] Adjust close icon vertical centering
- [x] Center arrow icon of token dropdown
- [x] Remove spacing after last row in delegation center
- [x] Center undelegated text in stake pool card
- [x] Only show CoinGecko info message if currencies have loaded
- [x] Vertically center "redelegate" button text in staking card
- [x] For empty wallets disable filter button
- [x] Clear all text if selected and typing in send dialog
- [x] Move hardware wallet disconnect message inside text field
- [x] Remove scrollbar in asset name popover
- [x] Saturation progress bar fixed width
- [x] Adjust colored line in stakepool tile
- [x] Correct 'cardano' and 'lightBlue' theming for software update 'x' button
- [x] Correct 'cardano' and 'lightBlue' theming for cardano node status 'x' button and scrollbar color
- [x] Update line-height for wallet creation/restore/import/HW buttons
- [x] Update padding in restore wallet dialog stepper
- [x] Update font used for staking pool tiles
- [x] Update font used for staking pools headings
- [x] Update positioning of 'back to top' button in stake pools delegation dialog
- [x] Update disabled select in redeem dialog
- [x] Fix glow effect in stake pools delegate dialog
- [x] Fix alignment of loading indicator in estimated fees input

## Screenshots

- Screenshot

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -228,7 +228,7 @@
     "react-intl": "2.7.2",
     "react-lottie": "1.2.3",
     "react-markdown": "4.3.1",
-    "react-polymorph": "1.0.0-next.1",
+    "react-polymorph": "1.0.1-rc.1",
```
[the remark this answers is not in the document]
Comment (mchappell) [reply package.json:231]: Once this is tested in Daedalus and everyone is satisfied that it works, I will release [1.0.1](https://github.com/input-output-hk/react-polymorph/pull/190), publish to NPM, and update the package here to use 1.0.1

File: source/renderer/app/components/layout/TopBar.scss [hunk 2]
```diff
@@ -36,14 +36,16 @@
       .walletName {
         -webkit-box-orient: vertical;
         color: var(--theme-topbar-wallet-name-color);
+        display: -webkit-box;
         font-size: 21px;
         -webkit-line-clamp: 1;
         line-height: 25px;
         margin-bottom: 4px;
-        max-width: 320px;
+        max-width: 24vw;
```
[the remark this answers is not in the document]
Comment (danielmain) [reply source/renderer/app/components/layout/TopBar.scss:44]: I did not know this rule. Should he use percentages ?

File: source/renderer/app/components/staking/delegation-center/helpers.js [hunk 3]
```diff
@@ -22,10 +22,7 @@ export const generateFieldPanel = (labels: any, values: any, index: number) => {
   const labelStr = labels[index];
   const valueStr = value.toString();
   let zeroValues = '';
-  if (
-    (index === 1 && valueStr.length < values[index + 1].toString().length) ||
-    (index === 0 && valueStr.length < EPOCH_MAX_LENGTH)
-  ) {
+  if (index === 1 && valueStr.length < values[index + 1].toString().length) {
```
Comment (DominikGuzei) [inline source/renderer/app/components/staking/delegation-center/helpers.js:25]: Are we sure that the condition `index === 0 && valueStr.length < EPOCH_MAX_LENGTH` is not needed anymore?

**Review by DominikGuzei — CHANGES_REQUESTED** [review]

File: source/renderer/app/components/staking/delegation-center/helpers.js (hunk 3 above)
Comment (tomislavhoracek) [reply source/renderer/app/components/staking/delegation-center/helpers.js:25]: @mchappell  I agree with D, we need to check that

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomislavhoracek) [reply package.json:231]: @mchappell 👌  But please ask at least one QA to double check PR after publishing 1.0.1. and applying to Daedalus

[the remark this answers is not in the document]
Comment (mchappell) [reply package.json:231]: @tomislavhoracek Will do, that can be the final check.

File: source/renderer/app/components/hardware-wallet/HardwareWalletStatus.js [hunk 4]
```diff
@@ -22,7 +22,8 @@ const messages = defineMessages({
   },
   connecting_failed: {
     id: 'wallet.hardware.deviceStatus.connecting.failed',
-    defaultMessage: '!!!Unable to detect your hardware wallet device. ',
+    defaultMessage:
+      '!!!Disconnect and reconnect your device to restart the process',
```
[the remark this answers is not in the document]
Comment (mchappell) [reply source/renderer/app/components/hardware-wallet/HardwareWalletStatus.js]: Space after dot wasn't needed as this now shows inside the input.

File: source/renderer/app/components/layout/TopBar.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (mchappell) [reply source/renderer/app/components/layout/TopBar.scss]: Adjusted to use a percentage. this now handles sidebar open/closed.
Tested using the widest characters possible for wallet name.

**Comment by gabriela-ponce** [comment]
@mchappell I reviewed the document and the previous issues again and almost everything is fixed, great job.

There are a few that still reproduce and some to be discussed with @alexander-rukin .
I updated [the document ](https://docs.google.com/document/d/1ai_8aqJfFhBIbsLA8I66GvLQ9u0EZamNM6v4KLbr8p8/edit?usp=sharing)and we can have a call tomorrow to check those.
Here's an issue that's not included in the document that still reproduces:
- The tooltip for unformatted values of Native tokens looks wrong with big numbers. Check [this](https://drive.google.com/file/d/1a9qCJd3EOJn_rdX1mhqILrCwtn8ju0sJ/view?usp=sharing) and [this](https://drive.google.com/file/d/1ffNTmhbLA0ampUtYcW-Xvc_ca6j9LJKB/view?usp=sharing) screenshot (on the last one, there is no space between the : and the number).

**Comment by alexander-rukin** [comment]
> * The tooltip for unformatted values of Native tokens looks wrong with big numbers. Check [this](https://drive.google.com/file/d/1a9qCJd3EOJn_rdX1mhqILrCwtn8ju0sJ/view?usp=sharing) and [this](https://drive.google.com/file/d/1ffNTmhbLA0ampUtYcW-Xvc_ca6j9LJKB/view?usp=sharing) screenshot (on the last one, there is no space between the : and the number).

Added this issue to the next [doc](https://docs.google.com/document/d/1lLnYnbIAxwcS2Sag6oWuEyhzdyPImd6oaSWa1qH95o4/edit?usp=sharing)

**Comment by mchappell** [comment]
> @mchappell I reviewed the document and the previous issues again and almost everything is fixed, great job.
> 
> There are a few that still reproduce and some to be discussed with @alexander-rukin . I updated [the document ](https://docs.google.com/document/d/1ai_8aqJfFhBIbsLA8I66GvLQ9u0EZamNM6v4KLbr8p8/edit?usp=sharing)and we can have a call tomorrow to check those. Here's an issue that's not included in the document that still reproduces:
> 
> * The tooltip for unformatted values of Native tokens looks wrong with big numbers. Check [this](https://drive.google.com/file/d/1a9qCJd3EOJn_rdX1mhqILrCwtn8ju0sJ/view?usp=sharing) and [this](https://drive.google.com/file/d/1ffNTmhbLA0ampUtYcW-Xvc_ca6j9LJKB/view?usp=sharing) screenshot (on the last one, there is no space between the : and the number).

Unformatted values native token moved to DDW-807

**Review by gabriela-ponce — APPROVED** [review]
Great job 🎉

**Review by dmitrii-gaico — APPROVED** [review]

**Pull request merged**